lib/net.js

```diff
@@ -1130,8 +1130,12 @@ function _listen(handle, backlog) {
   return handle.listen(backlog || 511);
 }
 
-var createServerHandle = exports._createServerHandle =
-    function(address, port, addressType, fd) {
+const createServerHandle = exports._createServerHandle = function(
+  address,
+  port,
+  addressType,
+  fd
+) {
```

Comment:
is this a specific case where the final `)` is supposed to be dropped to the next line? b/c this isn't happening most everywhere else.